src/suews/src/suews_phys_stebbs.f95

```diff
@@ -724,168 +728,7 @@ SUBROUTINE stebbsonlinecouple( &
 
             !       !
             IF (flginit == 0) THEN
```

Comment:
If this is part of the initialisation stage, we should probably move it out of the main SUEWS calculation.